sonar-flex-plugin/src/main/java/org/sonar/plugins/flex/cobertura/CoberturaReportPasrer.java

```diff
@@ -69,17 +68,23 @@ public void stream(SMHierarchicCursor rootCursor) throws XMLStreamException {
     }
   }
 
-  private static void collectPackageMeasures(SMInputCursor pack, SensorContext context, Project project, ModuleFileSystem fileSystem) throws XMLStreamException {
+  private static void collectPackageMeasures(SMInputCursor pack, SensorContext context, FileSystem fileSystem) throws XMLStreamException {
     while (pack.getNext() != null) {
       Map<String, CoverageMeasuresBuilder> builderByFilename = Maps.newHashMap();
       collectFileMeasures(pack.descendantElementCursor("class"), builderByFilename);
+      FilePredicates predicates = fileSystem.predicates();
 
       for (Map.Entry<String, CoverageMeasuresBuilder> entry : builderByFilename.entrySet()) {
-        Resource resource = getResourceForFileRelativeName(entry.getKey(), fileSystem, project);
 
-        if (resource != null) {
+        String fileName = (entry.getKey().startsWith(File.separator) ? entry.getKey() : File.separator + entry.getKey());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove those useless parentheses. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessParenthesesCheck)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add parentheses to make the operator precedence explicit. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS864)